src/components/Keyword/KeywordList/index.js

```diff
@@ -0,0 +1,329 @@
+                if(item.url.includes(AOUMIR)){
+                    return item;
+                }
+            })
+
+            setList(filterList);
+        }
+
+    },[keywordList]);
+
+    const onClickAllSelect = useCallback(()=>{
+
+        setCheckAll((prev)=>!prev);
+        
+    },[list,checkAll]);
+
+    const onClickReadNotification = useCallback(()=>{
+        setReadNotification((prev)=>!prev);
+        setNotReadNotification(false);
+    },[notReadNotification]);
+
+    const onClickNotReadNotification = useCallback(()=>{
+        setNotReadNotification((prev)=>!prev);
+        setReadNotification(false);
+    },[readNotification]);
+
+    const onChangeKeywordSearch = useCallback((e)=>{
+        setKeywordSearch(e.target.value);
+    },[keywordSearch]);
+
+    const onClickSearch = useCallback((e)=>{    
+        
+        const filterList = list.filter((item)=>{
+            
+            if(`${item.title}`.includes(`${keywordSearch}`)){
+                return item;
+            }
+        });
+
+        setList(filterList);
+        setKeywordSearch('');
+        inputSearch.current.focus();
+    },[list,keywordSearch]);
+
+    const onClickGoStore = useCallback(()=>{
+        setGoStore(true);
+    },[]);
+
+    const onClickDeleteList = useCallback(()=>{
+        setDeleteList(true);
+    },[]);
+
+    const onClickCheckSome = useCallback((id)=>{
+
+        let newCheckListId = [...checkListId];
+
+        if(checkListId.includes(id)){
+            
+            newCheckListId = checkListId.filter((item)=>{
+                if(item !== id){
+                    return item;
+                }
+            })
+
+            setCheckListId(newCheckListId);
+        }else{
+            newCheckListId.push(id);
+            setCheckListId(newCheckListId);
+        }
+
+    },[checkListId]);
+
+    const onClickReadItem = useCallback((id,isRead)=>{
+        if(!isRead){
+            dispatch(readKeywordItem(id));
+        }
+    },[])
+
+
+    useEffect(()=>{
+        if(list.length===0){
+            return;
+        }
+
+        if(notReadNotification){
+            const filterList = keywordList.filter((item)=>{
+                return item.isRead === false;
+            })
+            setList(filterList);
+        }else if(readNotification){
+            const filterList = keywordList.filter((item)=>{
+                return item.isRead === true;
+            })
+            setList(filterList);
+        }else{
+            setList(keywordList);
+        }
+    },[notReadNotification,readNotification]);
+
+    useEffect(()=>{
+
+        if(list.length===0){
+            return;
+        }
+
+        if(goStore){
+            if(checkAll){
+                alert('모든 목록 보관');
+                setCheckAll(false);
+                setGoStore(false);
+
+                keywordList.forEach((item)=>{
+                    const id = item.id
+                    dispatch(moveKeywordItem(id));
```

Comment:
굳이 식별자 선언이 필요 없어보이네요

```suggestion
                    dispatch(moveKeywordItem(item.id));
```